package.json

```diff
@@ -5,12 +5,14 @@
   "main": "index.js",
   "scripts": {
     "benchmark": "node benchmark",
-    "eslint": "eslint *.js benchmark spec",
-    "test-spec-watch": "mocha spec/*.spec.js -R spec --watch",
```

Comment:
I'd prefer we just document:

```hs
$ yarn test-node --watch
```

Rather than multiplying the number of permutations in `package.json:scripts`

Side note -- I believe the best normal way to view developing is "just run `yarn test-node`" and saving the browser tests for before a PR / when you're doing something that might drastically change from browser-to-browser. But since we're just doing mostly vanillajs, the normal happy path workflow should be fast.

Reply:
wouldn't it be `yarn test-node -- --watch`?

Reply:
It would be `npm run test-node -- --watch` or `builder run test-node -- --watch`.

However `yarn` optimizes away the need for both `run` and the "extra flags flag (`--`)" in most cases. In fact, if you _do_ use `--` then you get this helpful message:

```
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
```